packages/validate/src/ValidateMixin.js

```diff
@@ -261,6 +261,12 @@ export const ValidateMixin = dedupeMixin(
        * call. Situation B will occur after every call.
        */
       async validate({ clearCurrentResult } = {}) {
+        if (this.hasAttribute('disabled')) {
```

Comment:
could we use `this.disabled` here? as attributes are updated async

Reply:
Changed to this.disabled as suggested.

Reply:
nice tests are still green 💪